src/commonmark/en/content/user/configure-metadata.md

```diff
@@ -3946,19 +3948,22 @@ This way you can create a subset of options in an option set. The main purpose o
 
 2. Click the add button.
 
-3. In the **Name** field, type the name of the option group.
-
-4. In the **Option set** field, selected the option set that contains options you want to group.
+3. Fill in the form:
+    1. **Name**
+    2. **Short name**
+    2. **Code**
+    4. **Option set**
 
-5. Once an **Option set** is selected, you can select the **Options** you want to group.
+4. Once an **Option set** is selected, you can assign the **Options** you want to group.
 
-6. Click **save**.
+6. Click **Save**.
 
 ### Create or edit an option group set
 
 <!--DHIS2-SECTION-ID:create_option_group_set-->
 
 **Option group sets** allows you to categorise multiple **option groups** into a set.
+The main purpose of the option group set is to add more dimensionality to your captured data for analysis.
 > **Note**
```

Comment:
You need to make sure there is a blank line before this "> **Note**" so that the markdown will render properly
In general blank lines seem to be incredibly important in commonmark, in order to distinguish different block of text. (This is a bit of a weakness of the specification)